packages/perseus-editor/src/components/graph-locked-figures/locked-label-settings.tsx

```diff
@@ -0,0 +1,156 @@
+/**
+ * LockedLabelSettings is a component that allows the user to edit the
+ * settings of specifically a locked label on the graph within the
+ * Interactive Graph widget.
+ *
+ * Used in the interactive graph editor's locked figures section.
+ */
+import {
+    lockedFigureColors,
+    type LockedFigure,
+    type LockedFigureColor,
+    type LockedLabelType,
+} from "@khanacademy/perseus";
+import {View} from "@khanacademy/wonder-blocks-core";
+import {OptionItem, SingleSelect} from "@khanacademy/wonder-blocks-dropdown";
+import {TextField} from "@khanacademy/wonder-blocks-form";
+import {Strut} from "@khanacademy/wonder-blocks-layout";
+import {spacing, color as wbColor} from "@khanacademy/wonder-blocks-tokens";
+import {LabelLarge, LabelMedium} from "@khanacademy/wonder-blocks-typography";
+import {StyleSheet} from "aphrodite";
+import * as React from "react";
+
+import CoordinatePairInput from "../coordinate-pair-input";
+import PerseusEditorAccordion from "../perseus-editor-accordion";
+
+import ColorSelect from "./color-select";
+import LockedFigureSettingsActions from "./locked-figure-settings-actions";
+
+import type {LockedFigureSettingsCommonProps} from "./locked-figure-settings";
+
+export type Props = LockedLabelType &
+    LockedFigureSettingsCommonProps & {
+        /**
+         * Called when the props (coord, color, etc.) are updated.
+         */
+        onChangeProps: (newProps: Partial<LockedFigure>) => void;
+    };
+
+export default function LockedLabelSettings(props: Props) {
+    const {coord, color, size, text, onChangeProps} = props;
+
+    return (
+        <PerseusEditorAccordion
+            expanded={props.expanded}
+            onToggle={props.onToggle}
+            header={
+                <View style={[styles.row, styles.accordionHeaderContainer]}>
+                    <LabelLarge>
+                        Label ({coord[0]}, {coord[1]})
+                    </LabelLarge>
+                    <Strut size={spacing.xSmall_8} />
+                    {text !== "" && (
```

Comment:
Does something break if we render a `LabelLarge` with blank text? Or is it just bad for accessibility?

I wonder if we should default `text` to something like `(empty)` for display purposes, to keep the UI consistent.

Reply:
It just shows a little white circle because of the padding. It doesn't really matter, but it looked weird.